docs/content/en/docs/architecture/components/scheduler/_index.md

```diff
@@ -18,7 +18,7 @@ Instead, it uses
 the [Pod Scheduling Readiness K8s API](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness)
 to gate Pods until the required deployment checks pass.
 
-## Keptn Scheduling Gates for K8s 1.27 and above
+## Keptn Scheduling Gates for K8s 1.26 and above
```

Comment:
in 1.26, scheduling gates need to be enabled on k8s, as they are behind a feature flag, or are they enabled by default ?

Reply:
We enable it w/ 1.26: https://github.com/keptn/lifecycle-toolkit/blob/main/lifecycle-operator/chart/templates/deployment.yaml#L173

Reply:
no, we actually enable it with 1.27. the conditional says if less or equal 1.26 or !schedulinggates.enabled

Reply:
agree with @mowies, and that's just enabling it in Keptn, but what I meant here is the cluster. For example, if you are using KinD with k8s 1.26, you need to enable schedulingGates feature via config, see here https://github.com/keptn/lifecycle-toolkit/pull/632/files#diff-83ad539a565fa17dbd8dd6c23f74563225205b06e5e5a0fb14a4139e2974273f